fetchy/fetchyboi.ts

```diff
@@ -43,6 +43,9 @@ const extractAndStoreVaccineData = async (rawHtml: string) => {
     vaccineData.dataValidAsAtNzTimeIso
   );
 
+  if (existingData) {
+    console.log(diff(existingData.data, vaccineData));
```

Comment:
Does this log in github?

Reply:
yeh it should do - It was a debug thing though will remove

Reply:
Cool cool